### DISCUSSION TITLE
xstate for Svelte

### DISCUSSION THREAD

**Discussion by edbek** [statelyai/xstate#1733] [General]
Will there be xstate for Svelte?

**Comment by davidkpiano** [comment]
Here's a template: https://codesandbox.io/s/xstate-svelte-template-jflv1

**Comment by f-elix** [comment]
This library is also very well made for integration with Svelte: https://github.com/distolma/xstate-svelte#readme.

And here is an alternative from the Xstate docs: https://xstate.js.org/docs/recipes/svelte.html.
